frontend/public/components/namespace.jsx

```diff
@@ -247,14 +247,14 @@ class NamespaceDropdown_ extends React.Component {
     const onChange = newNamespace => dispatch(UIActions.setActiveNamespace(newNamespace));
 
     return <div className="co-namespace-selector">
-      <span>{model.label}:</span>
       <Dropdown
         className="co-namespace-selector__dropdown"
         menuClassName="co-namespace-selector__menu"
         noButton
         canFavorite
         items={items}
         title={title}
+        labelPrefix={model.label}
```

Comment:
Why not just make this part of title if we have no special styling?

```jsx
title={`${model.label}: ${title}`}
```

Reply:
Well, I did add a small amount of extra spacing between the colon and name, which I think helps the name standout better, but will change...

<img width="275" alt="screen shot 2018-07-11 at 10 57 38 am" src="https://user-images.githubusercontent.com/1874151/42580973-bece9c40-84f9-11e8-94e3-e33286dcf1e9.png">
<img width="268" alt="screen shot 2018-07-11 at 11 03 22 am" src="https://user-images.githubusercontent.com/1874151/42581129-110edccc-84fa-11e8-857c-ee00f4499013.png">

Reply:
pr rebased and updated